plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/replicateByte/case-5/case-5.uplc.expected

```diff
@@ -0,0 +1 @@
+(program 1.0.0 (con bytestring #ffffffff))
```

Comment:
This definitely shouldn't be happening, but I think that's because our Agda evaluator is dealing with `Word8` arguments incorrectly.  Updating with the `haskell-conformance` results should turn this into a failure.